### PR TITLE
[zh]Resync  concepts/services-networking/_index.md

### PR DESCRIPTION
**Pull request by mengjiao-liu**
[zh]Resync  concepts/services-networking/_index.md

en file： `content/en/docs/concepts/services-networking/_index.md`

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/_index.md [hunk 1]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+每一个 `Pod` 都有它自己的IP地址，这就意味着你不需要显式地在每个 `Pod` 之间创建链接，
+你几乎不需要处理容器端口到主机端口之间的映射。
+这将创建一个干净的、向后兼容的模型，在这个模型里，从端口分配、命名、服务发现、
+负载均衡、应用配置和迁移的角度来看，`Pod` 可以被视作虚拟机或者物理主机。
+
+Kubernetes 对所有网络设施的实施，都需要满足以下的基本要求（除非有设置一些特定的网络分段策略）：
+
+* 节点上的 Pod 可以不通过 NAT 和其他任何节点上的 Pod 通信
+* 节点上的代理（比如：系统守护进程、kubelet）可以和节点上的所有Pod通信
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:41]:
```suggestion
* 节点上的代理（比如：系统守护进程、kubelet）可以和节点上的所有 Pod 通信
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 2]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+备注：仅针对那些支持 `Pods` 在主机网络中运行的平台（比如：Linux）：
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:43]:
```suggestion
备注：对于支持在主机网络中运行 `Pod` 的平台（比如：Linux）：
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 3]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+* 那些运行在节点的主机网络里的 Pod 可以不通过 NAT 和所有节点上的 Pod 通信
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:45]:
```suggestion
* 运行在节点主机网络里的 Pod 可以不通过 NAT 和所有节点上的 Pod 通信
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 4]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+talk to other VMs in your project.  This is the same basic model.
+
+Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
+share their network namespaces - including their IP address and MAC address.
+This means that containers within a `Pod` can all reach each other's ports on
+`localhost`. This also means that containers within a `Pod` must coordinate port
+usage, but this is no different from processes in a VM.  This is called the
+"IP-per-pod" model.
+-->
+这个模型不仅不复杂，而且还和 Kubernetes 的实现廉价的从虚拟机向容器迁移的初衷相兼容，
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:60]:
```suggestion
这个模型不仅不复杂，而且还和 Kubernetes 的实现从虚拟机向容器平滑迁移的初衷相符，
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 5]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+如果你的工作开始是在虚拟机中运行的，你的虚拟机有一个 IP ，
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:61]:
```suggestion
如果你的任务开始是在虚拟机中运行的，你的虚拟机有一个 IP，
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 6]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+这样就可以和其他的虚拟机进行通信，这是基本相同的模型。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:62]:
```suggestion
可以和项目中其他虚拟机通信。这里的模型是基本相同的。
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 7]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+Kubernetes 的 IP 地址存在于 `Pod` 范围内 - 容器共享它们的网络命名空间 - 包括它们的 IP 地址和 MAC 地址。
+这就意味着 `Pod` 内的容器都可以通过 `localhost` 到达各个端口。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:65]:
```suggestion
这就意味着 `Pod` 内的容器都可以通过 `localhost` 到达对方端口。
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 8]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+这也意味着 `Pod` 内的容器都需要相互协调端口的使用，但是这和虚拟机中的进程似乎没有什么不同，
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:66]:
```suggestion
这也意味着 `Pod` 内的容器需要相互协调端口的使用，但是这和虚拟机中的进程似乎没有什么不同，
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 9]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+<!--
+How this is implemented is a detail of the particular container runtime in use.
+
+It is possible to request ports on the `Node` itself which forward to your `Pod`
+(called host ports), but this is a very niche operation. How that forwarding is
+implemented is also a detail of the container runtime. The `Pod` itself is
+blind to the existence or non-existence of host ports.
+-->
+如何实现这一点是正在使用的容器运行时的特定信息。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:77]:
```suggestion
如何实现以上需求是所使用的特定容器运行时的细节。
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 10]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+也可以在 `node` 本身通过端口去请求你的 `Pod`（称之为主机端口），
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:79]:
```suggestion
也可以在 `Node` 本身请求端口，并用这类端口转发到你的 `Pod`（称之为主机端口），
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 11]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+但这是一个很特殊的操作。转发方式如何实现也是容器运行时的细节。
+`Pod` 自己并不知道这些主机端口是否存在。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:81]:
```suggestion
`Pod` 自己并不知道这些主机端口的存在。
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 12]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
 -->
 
 Kubernetes 网络解决四方面的问题：
-- 一个 Pod 中的容器之间通过本地回路（loopback）通信。
+- 一个 Pod 中的容器之间[通过本地回路（loopback）通信](/zh/docs/concepts/services-networking/dns-pod-service/)。
 - 集群网络在不同 pod 之间提供通信。
-- Service 资源允许你对外暴露 Pods 中运行的应用程序，以支持来自于集群外部的访问。
-- 可以使用 Services 来发布仅供集群内部使用的服务。
\ No newline at end of file
+- [Service 资源](/zh/docs/concepts/services-networking/service/)允许你[对外暴露 Pods 中运行的应用程序](/zh/docs/concepts/services-networking/connect-applications-service/)，以支持来自于集群外部的访问。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/_index.md:94]:
```suggestion
- [Service 资源](/zh/docs/concepts/services-networking/service/)允许你
  [对外暴露 Pods 中运行的应用程序](/zh/docs/concepts/services-networking/connect-applications-service/)，
  以支持来自于集群外部的访问。
```

File: content/zh/docs/concepts/services-networking/_index.md [hunk 13]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+Note: For those platforms that support `Pods` running in the host network (e.g.
+Linux):
+
+* pods in the host network of a node can communicate with all pods on all
+  nodes without NAT
+-->
+## Kubernetes 网络模型   {#the-kubernetes-network-model}
+
+每一个 `Pod` 都有它自己的IP地址，这就意味着你不需要显式地在每个 `Pod` 之间创建链接，
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated. Thanks for catch!

File: content/zh/docs/concepts/services-networking/_index.md [hunk 14]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+你几乎不需要处理容器端口到主机端口之间的映射。
+这将创建一个干净的、向后兼容的模型，在这个模型里，从端口分配、命名、服务发现、
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Fixed.

File: content/zh/docs/concepts/services-networking/_index.md [hunk 15]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+负载均衡、应用配置和迁移的角度来看，`Pod` 可以被视作虚拟机或者物理主机。
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md [hunk 16]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+Kubernetes 对所有网络设施的实施，都需要满足以下的基本要求（除非有设置一些特定的网络分段策略）：
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md [hunk 17]
```diff
@@ -4,16 +4,92 @@ weight: 60
...
+
+* 节点上的 Pod 可以不通过 NAT 和其他任何节点上的 Pod 通信
```
[the remark this answers is not in the document]
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 1 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 12 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 11 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 10 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 9 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 8 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 7 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 6 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 2 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated. Thanks.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 3 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.Thanks.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 4 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

File: content/zh/docs/concepts/services-networking/_index.md (hunk 5 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/services-networking/_index.md]: Updated.

**Pull request merged**